src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NetCorePackageReferenceProjectProvider.cs

```diff
@@ -94,13 +103,16 @@ public NetCorePackageReferenceProjectProvider(IProjectSystemCache projectSystemC
             var fullProjectPath = vsProject.FullProjectPath;
             var unconfiguredProject = GetUnconfiguredProject(vsProject.Project);
 
+            var projectServices = new VsCoreProjectSystemServices(vsProject, _componentModel.Value);
```

Comment:
But we're anyways not going to use `DTE` there, It will always be using ScriptService only... But I think i see your point, it will be better to keep them separate.